src/components/challengeWidget/challengeWidget.js

```diff
@@ -32,6 +32,14 @@ const ChallengeWidget = ({ title, labels, linkTo = "#" }) => (
       <span className="marker">0</span>
       <span className="marker">3</span>
       <button className="currentPeriod">+</button>
+      <div className="months">
```

Comment:
I believe `.months` should not be its own `<div>` but should be inside the individual markers instead, that's where they logically belong to.